plugins/auth-backend/src/lib/flow/authFlowHelpers.ts

```diff
@@ -69,6 +69,14 @@ export const postMessageResponse = (
   res.end(`<html><body><script>${script}</script></body></html>`);
 };
 
+/** @public */
+export const redirectMessageResponse = (
+  res: express.Response,
+  redirectUrl: string,
+) => {
+  res.redirect(redirectUrl);
```

Comment:
honestly this probably doesn't need a function 😅. The symmetry is nice but I don't think it's enough of a benefit